content/docs/operating/security.md

```diff
@@ -177,6 +182,38 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
+
+In the future, server-side TLS support will be rolled out to the different
+projects under the Prometheus organization. Those projects include Prometheus,
+Alertmanager, Pushgateway and the official exporters.
+
+The golang projects will share the same TLS library, which will be based on the
+Golang vanilla TLS library. We keep [Golang's default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+support only TLSv1.2 and higher.
+
+Research will be made later on the non-golang exporters to see how to best
```

Comment:
Java may end up being weird (e.g. how do you only support 1.2+ and the various client auth options in a JVM that doesn't support it? Will it even compile?), so I'd phrase it more as "TLS will be added to Java exporters in the future".

We've no Python or Ruby exporters, so we can limit this to Java.

Reply:
Rephrased